### ISSUE TITLE
Duplicate classes are removed during compilation

### ISSUE THREAD

**Issue by CrshOverride** [ionic-team/stencil#217]
**Resources:**
Before submitting an issue, please consult our [docs](https://stenciljs.com/).

**Stencil version:** (run `npm list @stencil/core` from a terminal/cmd prompt and paste output below):

```
@stencil/core@0.0.6-5
```

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://stencil-worldwide.slack.com

**Current behavior:**
Compiling a component containing the following tsx:

```tsx
<div class='ui middle aligned center aligned grid'>
</div>
```

Produces the resulting markup:

```html
<div class='ui middle aligned center grid'>
</div>
```

**Expected behavior:**
While the HTML specification says that duplicate classes should be ignored, the compiler should still emit markup matching that of the tsx definition.

**Steps to reproduce:**
1. Start a new Stencil app.
2. Use the component below.
3. Run application at the console and view the markup.

**Related code:**

```tsx
import { Component, Prop } from '@stencil/core';

@Component({
  tag: 'my-div',
})
export class MyDiv {
  render() {
    return (
      <div class='ui middle aligned center aligned grid'>Hello!</div>
    );
  }
}
```

**Other information:**
This bug is most evident to those of us using Semantic UI for styling.

**Comment by jthoms1** [comment]
I think we can update the runtime to look at a class and if it is a string not an object then leave it alone.

**Comment by CrshOverride** [comment]
@jthoms1 I was looking through the compiler last night trying to figure out where the transpiled `tsx` file was munged and converted to the `c: {}` syntax but wasn't able to find it. Any pointers?

**Comment by jthoms1** [comment]
This should be fixed with commit 87a8b0b4a4ef341d66d2da1fd5a1abbfed9d8626. @CrshOverride you can take a look at the commit to familiarize yourself with the code changes.  We are no longer doing these changes in the compiler and it now occurs in the browser.
